Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM quay.io/konflux-ci/konflux-test:v1.4.38@sha256:c306aa4b764fcade1cbea8b8f7b6166e3a1289f56e03be99f669b9aaf7a92363 as konflux-test
 FROM registry.access.redhat.com/ubi9/ubi-minimal:9.6-1754000177
-
+#FROM registry.access.redhat.com/ubi9/ubi:9.6-1756915113
+#RUN rpmkeys --import /etc/pki/rpm-gpg/RPM-GPG-KEY-* --root "${NEWROOT}"
+#RUN ls /cachi2/output/deps
+#RUN rpmkeys --import /etc/pki/rpm-gpg/RPM-GPG-KEY-* --root "/cachi2/output/deps/rpm"
+#RUN dnf install -y "--installroot=${NEWROOT}" bash
 
 ENV POLICY_PATH="/project"
 # Install required packages
-RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
-    microdnf -y --setopt=tsflags=nodocs install \
+RUN microdnf -y --setopt=tsflags=nodocs install \
```

Comment:
Since you're using microdnf and properly filtering out docs packages you might also be interested in `--setopt=install_weak_deps=False`